usingcurl/verbose/writeout.md

```diff
@@ -70,7 +70,7 @@ Some of these variables are not available in really old curl versions.
 | `http_code`     | The former variable name for what is now known as `response_code`.                               |
 | `http_connect`   | the numerical code that was found in the last response (from a proxy) to a curl CONNECT request. |
 | `http_version`   | The HTTP version that was used.                                                                  |
-| `json`           | all write-out variables as a single JSON object.                                                 |
+| `json`           | all write-out variables as a single JSON object. ([Introduced in 7.72.0](https://github.com/curl/curl/pull/4870)) |
```

Comment:
I don't think we need the link to the PR. We don't have that anywhere else. We try to keep the book "simple"